.github/workflows/e2e-upgrade.yml

```diff
@@ -182,7 +182,7 @@ jobs:
           awsOpenSearchDomain: ${{ secrets.AWS_OPENSEARCH_DOMAIN }}
           awsOpenSearchUsers: ${{ secrets.AWS_OPENSEARCH_USER }}
           awsOpenSearchPwd: ${{ secrets.AWS_OPENSEARCH_PWD }}
-          selfManagedInfra: "false"
+          clusterCreation: "cli"
```

Comment:
I think we should run an upgrade test on the terraform provider

Reply:
I don't think we currently support this (without a decent amount of extra work) as you would imagine it.
The main "upgrade test" is run by a [Go test](https://github.com/edgelesssys/constellation/tree/main/e2e/internal/upgrade), which expects to use a CLI binary.
We *could* create the initial Constellation cluster using Terraform, but I'm not sure if this would assert anything helpful